instrumentation/resources/library/src/main/java/io/opentelemetry/instrumentation/resources/ProcessResource.java

```diff
@@ -23,13 +23,13 @@
 public final class ProcessResource {
 
   // copied from ProcessIncubatingAttributes
-  private static final AttributeKey<List<String>> PROCESS_COMMAND_ARGS =
+  static final AttributeKey<List<String>> PROCESS_COMMAND_ARGS =
       AttributeKey.stringArrayKey("process.command_args");
-  private static final AttributeKey<String> PROCESS_COMMAND_LINE =
+  static final AttributeKey<String> PROCESS_COMMAND_LINE =
       AttributeKey.stringKey("process.command_line");
-  private static final AttributeKey<String> PROCESS_EXECUTABLE_PATH =
+  static final AttributeKey<String> PROCESS_EXECUTABLE_PATH =
       AttributeKey.stringKey("process.executable.path");
-  private static final AttributeKey<Long> PROCESS_PID = AttributeKey.longKey("process.pid");
+  static final AttributeKey<Long> PROCESS_PID = AttributeKey.longKey("process.pid");
```

Comment:
I noticed the HostResource attribute constants were package private so I made this change so that I could access them in the test as well. If this is undesirable I can just hard code the Attribute in the test

Reply:
I think these can be private since they don't seem to be used outside of the class. If you need to use them in the tests then you should use `ProcessIncubatingAttributes` (we don't want to add the incubating semconv jar as dependency to libraries that we publish, but using it elsewhere is totally fine).

Reply:
thank you for the help and note about the incubating jars, I will keep that in mind going forward 🙏